src/ol/style/iconimagecache.js

```diff
@@ -91,3 +90,13 @@ ol.style.IconImageCache.prototype.set = function(src, crossOrigin, color, iconIm
   this.cache_[key] = iconImage;
   ++this.cacheSize_;
 };
+
+
+/**
+ * Set cache max size.
+ * @param {number} maxCacheSize Cache max size.
+ */
+ol.style.IconImageCache.prototype.setMaxCacheSize = function(maxCacheSize) {
```

Comment:
What about naming this nicer (`setSize`) and adding an `@api` annotation and some more verbose documentation?